html/components/SimpleHtml.js

```diff
@@ -25,6 +25,24 @@ import getEmptyObjectKeys from '../services/getEmptyObjectKeys';
 import isValidVideoFormat from '../services/isValidVideoFormat';
 import Image from './Image';
 
+const UNSPLASH_IMAGE_CONTAINER_ID = 'unsplash-image-container';
+
+function alterUnsplashImagesNode(node) {
+  const styleAttrib = _.get(node, 'attribs.style', '').trim();
+
+  if (node.name === 'div' && node.attribs.id === UNSPLASH_IMAGE_CONTAINER_ID) {
+    const imageContainerWithMargins = {
+      attribs: {
+        style: `${styleAttrib}margin-top: 15px; margin-bottom: 30px;`,
+      },
+    };
+    const resolvedNode = _.merge(node, imageContainerWithMargins);
+    return resolvedNode;
```

Comment:
I can't recall anymore for certain, but I believe we agreed to do spacing before `return` statements.

Reply:
Oopsie, ty